tests/bud.bats

```diff
@@ -7313,6 +7313,15 @@ _EOF
   local_head_hash=$output
   run_buildah run testctr -- sh -c 'cd podman-tag && git ls-remote --tags origin v5.0.0^{} | cut -f1'
   assert "$output" = "$local_head_hash"
+
+  cat > $contextdir/Dockerfile << _EOF
+FROM alpine
+RUN apk add git
```

Comment:
The image shouldn't need `git` installed for `ADD` to work, so this step is unnecessary.  It can be `FROM scratch` if it doesn't need to run anything after the content has been added.